controllers/utils/routeStatic.js

```diff
@@ -28,6 +33,28 @@ let servePage = (page, router) => {
     });
 };
 
+let serveCmsPage = (page, sectionId, router) => {
+    router.get(page.path, (req, res) => {
+        const renderPage = content => {
+            res.render('pages/legacy', {
+                title: content.title,
+                content: content
+            });
+        };
+
+        contentApi
+            .getLegacyPage(req.i18n.getLocale(), sectionId + page.path)
+            .then(response => get(response, 'data.attributes', []))
```

Comment:
I've been moving this kind of stuff into the content API method, and being explicit about params, so you'd end up with something like:

```
contentApi
  .getLegacyPage({
    locale: req.i18n.getLocale(),
    path: sectionId + page.path
  })
  .then(content => {
    renderPage(content);
  });
```